api/reporting/schemas/input/descriptions/user.yaml

```diff
@@ -0,0 +1,60 @@
+---
+user:
+  about_you: User's description of themselves.
```

Comment:
about_you should be like User's description of their research background, experience and research interests.